tests/test_lab/test_gates_fock.py

```diff
@@ -210,6 +217,43 @@ def test_raise_interferometer_error():
         RealInterferometer(num_modes=num_modes, modes=modes)
 
 
+@given(phase_stdev=medium_float.filter(lambda x: x > 0))
+def test_phasenoise_creates_dm(phase_stdev):
+    """test that the phase noise gate is correctly applied"""
+    assert (Coherent(1.0) >> PhaseNoise(phase_stdev))._dm is not None
+    assert (Fock(10) >> PhaseNoise(phase_stdev))._dm is not None
+
+
+@given(phase_stdev=medium_float.filter(lambda x: x > 0))
+def test_phasenoise_symmetry(phase_stdev):
+    assert (Fock(1) >> PhaseNoise(phase_stdev)) == Fock(1)
+    settings.AUTOCUTOFF_MIN_CUTOFF = 100
+    assert (Thermal(1) >> PhaseNoise(phase_stdev)) == Thermal(1)
+    settings.AUTOCUTOFF_MIN_CUTOFF = 1
```

Comment:
Would be nice to have this cleaning line as part of a fixture since it's repeated across tests.